content/applications/websites/website/configuration/multi_website.rst

```diff
@@ -2,316 +2,164 @@
 Multiple websites
 =================
 
-.. image:: multi_website/multi_website04.png
-  :align: center
+Odoo allows you to create multiple websites from the same database. This can be useful, for example,
+if you have multiple brands operating under your organization, or to create separate websites for
+different products/services, or different audiences. In these cases, having different websites can
+help avoid confusion and make it easier to tailor your digital outreach strategies and reach your
+target audience.
+
+Each website can be designed and configured independently with its own :doc:`domain name
+<../../../../administration/maintain/domain_names>`, theme, pages,
+menus, :doc:`languages <translate>`, :doc:`products <../../ecommerce/managing_products/products>`,
+assigned sales team, etc. They can also :ref:`share contents and pages
```

Comment:
So I researched this and in this case, you're right, I should've used the unaccountable version. 

But (and just an FYI in case you're interested) it can also be used in the plural form when you refer to something within a container, like a pan, a drawer or a grocery bag for example
https://www.merriam-webster.com/dictionary/content
https://www.collinsdictionary.com/dictionary/english/content

Reply:
cool, thanks!!!